deepchem/feat/molecule_featurizers/roberta_tokenizer.py

```diff
@@ -0,0 +1,74 @@
+from deepchem.feat import MolecularFeaturizer
+from deepchem.utils.typing import RDKitMol
+from typing import Dict, List
+try:
+  from transformers import RobertaTokenizerFast
```

Comment:
How's the debugging going on this? Are we able to install `transformers` without pytorch/tensorflow/etc safely? If so let's just move it into common as a base dependency

Reply:
It seems like transformers requires either pytorch and/or tensorflow (one or the other) to work - so we should probably not move it into common?

Reply:
Hmm, ok, let's do this. Let's mark this a PyTorch only feature and just move forward. So for the unit tests you'll need to add a `@pytest.mark.torch` annotation so the tests only run for pytorch. You'll also need import guards like the other torch models do

Reply:
Sounds good - transformers is listed as a requirement in `env_torch.yml` but not `env_torch_cpu.yml` - should I add it there too?

Reply:
I'm not 100% sure; you might have to experiment and see what gets tests passing, but adding it there sounds reasonable to me!

Reply:
cool - added it to the torch cpu